### DISCUSSION TITLE
Keydown/keyboard events to line chart

### DISCUSSION THREAD

**Discussion by jeny-feldman** [plouc/nivo#2235] [Q&A]
I want to add keyboard events, so the user will able to navigate between the points in the line with the keyboard.
Is there a support for that?

**Comment by SebasQuirogaUCP** [comment]
There is a stateful variable _setCurrentPoint_ used internally for onMouse & onClick events, but there is not a prop to set it externally.
It sounds like a good new feature to add, what about opening a new request  issue?
